contracts/WitnetRequestsBoard.sol

```diff
@@ -63,6 +63,15 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
   // Event emitted when a result proof is posted
   event PostedResult(address indexed _from, uint256 _id);
 
+  // Event myass
+  event MyAss(uint256 _id);
+
+  // Event myass
+  event MyAss3(uint256 _id);
+
+  // Event myass
+  event MyAss2(bool id);
+
```

Comment:
I think you forgot to remove these events :rofl:

Reply:
hahahahahahhahahahahahha